tests/integration_test/cors_integration_test.go

```diff
@@ -537,6 +537,7 @@ func TestServiceControlRequestWithAllowCors(t *testing.T) {
 					URL:               "/bookstore/shelves?key=api-key",
 					ApiKey:            "api-key",
 					ApiMethod:         "1.echo_api_endpoints_cloudesf_testing_cloud_goog.CorsShelves",
+					ApiName:           "1.echo_api_endpoints_cloudesf_testing_cloud_goog",
```

Comment:
Check API version here too?

Reply:
There's a few more tests where this isn't checked in other integration test files, can you also check there? Or is it not needed?

Reply:
Those methods don't have apiVersion because they aren't under Apis.Methods(only Apis define the version) but generated by  HttpRules.